### ISSUE TITLE
Protecting Like / Dislike with auth?

### ISSUE THREAD

**Issue by Mirv** [jatwell93/rubyproject#47]
I prose like/dislike & review should be protected by login or something to prevent people from manipulating popularity type situations - though honestly - one good strategy to build the site is leave as much open as possible.

**Comment by jatwell93** [comment]
Good point on both. What way are you leaning more towards? I was thinking keep like open and review under auth?

**Comment by Mirv** [comment]
I think for now the only changes I would make to the like/dislike is perhaps fleshing out a full fledged controller for the model you have already - which would allow reversal of a like.  It also, might open the door to interesting things such as Love/Like/Dislike/Hate instead of boolean.

**Comment by jatwell93** [comment]
That would be great. I've been doing a lot of searching on the reversal aspect and have come up with some ideas for the code, but not quite sure how to implement them along with the counting system etc.

**Comment by Mirv** [comment]
So actually - like/dislike HAS to be by registered users....otherwise there's no ID to stick in there - unless you goto a click button type situation where it just totals every time & someone can just keep clicking it!
